allure-model/src/main/java/ru/yandex/qatools/allure/config/AllureConfig.java

```diff
@@ -70,6 +70,9 @@
     @Property("allure.tests.management.pattern")
     protected String tmsPattern = "%s";
 
+    @Property("allure.are.testng.parameters.disabled")
```

Comment:
let's change it to `allure.testng.parameters.enabled` with `true` as default value.